packages/@react-native-windows/tester/src/js/examples/View/ViewExample.windows.js

```diff
@@ -517,6 +519,71 @@ exports.examples = [
       return <DisplayNoneStyle />;
     },
   },
+{
+  title: '`display:none` style with input',
+    render(): React.Node {
+    type Props = $ReadOnly<{||}>;
+  type State = {|
+    displayNone: boolean,
+      displayNone2: boolean,
+        text: string,
+          text2: string,
+      |};
+
+class DisplayNoneVisibilityExample extends React.Component<Props, State> {
+  state = {
+    displayNone: false,
```

Comment:
nit: you could also just store the string value here, instead of the boolean

Reply:
I switched to string for a moment, but then I'd have to make the switch press handlers messier :' ( I think I might just keep this if that's good.